SQL1/Homework1-SQL.sql

```diff
@@ -0,0 +1,173 @@
+CREATE DATABASE ZooManagement5;

```

Comment:
Not sure why the database and the table names have a number as a suffix, it would be easier to read if those numbers weren't here.